pybossa.js

```diff
@@ -102,6 +102,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         });
     }
 
+    function _releaseCategoryLocks(projectname, taskId) {
+        var data = {
+            'projectname': projectname
+        };
+        return $.ajax({
+            type: 'POST',
+            url: url + 'api/task/' + taskId + '/release_category_locks',
```

Comment:
If `release_category_lock` is supposed to release all reservations, what is the purpose of passing task_id to the API?

Reply:
Good question. The reason we need task_id because we will need it to get the *same* category of this task: [```reserve_key = get_reserve_task_key(task_id)```](https://github.com/bloomberg/pybossa/pull/731/files#diff-f7eb1704644fb813492849abfa5e3f07f4c3cd76c4f60088485e5ffe204b5024R595)  ,that is, we want to find out the keys like reserve_task:project:1:category:fruit:apple:user:1 by the task id.